openedx/features/survey_report/api.py

```diff
@@ -35,3 +37,45 @@ def generate_report() -> None:
     )
 
     survey_report.save()
+    return survey_report.id
+
+def send_report(report_id: int) -> None:
```

Comment:
What about:
send_report_to_external_service
send_report_to_service
?

Reply:
@mariajgrimaldi This is like the generate_report method, is just the action, the answer to "where" depends of the implementation, and maybe that will be a setting in a future, similar to backends or repositories.